client/src/store/historyStore/historyChangedItemsStore.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Monitors and requests the summary of recently changed history items and commits the
+ * results to the history items and other attached stores. The initial update time threshold
+ * corresponds to the module import time. Once the start action is called, continuous requests
+ * are submitted while the threshold is being increased by the throttle period and the request
+ * response time for every consecutive request.
+ */
+import { urlData } from "utils/url";
+
+let fetching = false;
+let lastDate = new Date();
+const limit = 100;
+const throttlePeriod = 3000;
+
+const actions = {
+    fetchHistoryChangedItems: async ({ commit, dispatch }, { historyId }) => {
+        const params = `limit=${limit}&q=update_time-ge&qv=${lastDate.toISOString()}`;
```

Comment:
```suggestion
        const params = `limit=${limit}&q=update_time-ge&qv=${encodeURIComponent(lastDate.toISOString())}`;
```
that fixes the update time issue for me. However the API now keeps returning the updated dataset, even though the update time increments.

Reply:
Nope, not sure what I saw there, with the suggestion it seems to work.

Reply:
I tried it and it works for me too. Thx for the fix.

Reply:
I think @mvdbeek is right here; the colons in the iso date string should be encoded -- it doesn't look like this got applied?